README.md

```diff
@@ -29,6 +29,7 @@ let props = {
   optionalMedia: ElementPropTypes.media,
   optionalEditorFull: ElementPropTypes.editorFull,
   optionalEditorMinimal: ElementPropTypes.editorMinimal,
+  optionalReadonly: ElementPropTypes.readonly,
```

Comment:
Will you be adding the required version to the docs as well?

Reply:
Also, how about `ReadOnly` and `readOnly` for consistency with whats in the other files?

Reply:
@jason-covington As the property is not to be editable, we don't need a required. I will fix the casing tho. Thanks.